src/test/java/com/puppycrawl/tools/checkstyle/PropertyCacheFileTest.java

```diff
@@ -92,7 +93,9 @@ public void testCtor() {
     @Test
     public void testInCache() throws IOException {
         final Configuration config = new DefaultConfiguration("myName");
-        final String filePath = File.createTempFile("junit", null, temporaryFolder).getPath();
+        final String uniqueFileName = "junit_" + UUID.randomUUID() + ".java";
+        final String filePath =
+                Files.createFile(temporaryFolder.toPath().resolve(uniqueFileName)).toString();
```

Comment:
=> `new File(temporaryFolder, uniqueFileName).getAbsolutePath()` or tell me why not.